docs/types.propDefinition.md

```diff
@@ -357,7 +357,7 @@ observable property.  These behaviors can be specified with as an `Object`, `Str
 
   const Person = DefineMap.extend( {
     fullName: {
-      set(newValue) {
+      set(newVal) {
```

Comment:
I’d use `newValue` everywhere instead of `newVal`

Reply:
Fixed.